pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>io.openraven.magpie</groupId>
   <artifactId>magpie-api</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>0.2.1-SNAPSHOT</version>
```

Comment:
No need to update this, no release has been made!  As long as you `mvn install` locally you'll be fine.

Reply:
reverted